inheritance/src/main/java/inheritance/Theater.java

```diff
@@ -0,0 +1,32 @@
+package inheritance;
+
+import java.util.ArrayList;
+
+public class Theater extends Business {
+  ArrayList<String> movies;
+
+  public Theater(String name, int stars){
+    super(name, stars);
+    this.movies = new ArrayList<>();
+  }
+
+  public void addMovie(String movie){
+    for(String m : this.movies){
+      if(movie != m){
```

Comment:
1. When comparing strings you should ALWAYS use `.equals()` instead of `!=`
2. This method will add `movie` to the `movies` list every time the conditional fails. You could have the same movie in the list many times!